### PR TITLE
Updating site for 2019

### PR DESCRIPTION
**Pull request by aschearer**
Preparing SIX website for a new year.

### CODE REVIEW

File: _data/social-media.yml [hunk 1]
```diff
@@ -9,4 +9,4 @@ Facebook:
 
 Eventbrite:
   svg: eventbrite
-  url: https://six-2018.eventbrite.com
+  url: https://six-2019.eventbrite.com
```
Comment (constancecchen) [inline _data/social-media.yml]: Nice job getting this page reserved already! I think usually we don't make a page until the lineup gets announced, so this is awesome and ahead of schedule.

If for whatever reason you don't want people finding the eventbrite page until you're ready, you can set `registrationOpen` to false in `_data/schedule.json`.

File: _includes/svgs/robot7.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...3.57" r="13.26"/><rect class="cls-3" x="104.11" y="167.16" width="21.43" height="8.96" rx="2.78" transform="translate(277.71 298.57) rotate(161.71)"/><rect class="cls-3" x="183.58" y="167.74" width="6.15" height="29.1" rx="2.56" transform="translate(257.17 433.49) rotate(-150)"/><path class="cls-5" d="M216.58,304.53a28.43,28.43,0,0,0-27.47,28c0,.55-.76,5,3.94,4.86,4.46-.11,3.1-4.32,3.1-4.83,0-11.37,11.39-21.15,20.43-21.19,12.14-.05,20.48,9.91,20.48,21.28,0,1.08-.55,5.16,4,5.16,3.47,0,3.42-4.23,3.42-5.28C244.48,317.77,234.29,304.14,216.58,304.53Z"/><rect class="cls-5" x="189.98" y="338.88" width="51.42" height="7.16" rx="2.74"/><rect class="cls-2" x="207.73" y="318.48" width="17.71" height="53.02" rx="7.03"/><ellipse class="cls-3" cx="193.41" cy="339.92" rx="8.49" ry="8.26"/><ellipse class="cls-3" cx="240.74" cy="341.33" rx="8.49" ry="8.26"/><path class="cls-3" d="M180.15,217.83a264.44,264.44,0,0,0-6.22,29.08,5,5,0,0,0,4.92,5.74h34.87a5,5,0,0,0,4.53-7l-13.37-29.51a5,5,0,0,0-2.11-2.3c-3-1.64-10.2-4.27-20.17.95A4.93,4.93,0,0,0,180.15,217.83Z"/><rect class="cls-5" x="202.93" y="224.3" width="4.6" height="16.59" rx="2.3" transform="translate(-74.94 99.46) rotate(-23.16)"/></g></g></svg>
```
Comment (constancecchen) [inline _includes/svgs/robot7.svg]: These look rad. Cool find!

File: _pages/about.html [hunk 3]
```diff
@@ -24,25 +24,37 @@
...
+
+      - name: Alex
+        svg: robot2
+        url: https://www.twitter.com/AlexSchearer
+
+      - name: Austin
         svg: robot4
+        url: https://www.linkedin.com/in/austin-maestre-82884054
+
+      - name: Constance
```
Comment (constancecchen) [inline _pages/about.html]: You can remove me if you like honestly :)

File: _data/schedule.json [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
   "startOfYear": false,
 
-  "applicationsOpen": false,
+  "applicationsOpen": true,
   "applicationsOpenDate": "May 1st",
-  "applicationsCloseDate": "June 9th",
+  "applicationsCloseDate": "June 8th",
 
-  "juryingInProgress": false,
+  "juryingInProgress": true,
```
Comment (constancecchen) [inline _data/schedule.json]: This should be false, right? Jurying hasn't begun yet?

File: _years/2018/index.md [hunk 5]
```diff
@@ -1,5 +1,17 @@
 ---
-layout: none
+title: SIX 2018
```
Comment (constancecchen) [inline _years/2018/index.md]: Super awesome job with this page and the thumbnail. I honestly think I might forgotten what it's like working with super intelligent self-starters who can get shit done without a ton of hand-holding. You guys rock :)

File: firestore.rules [hunk 6]
```diff
@@ -56,8 +56,8 @@ service cloud.firestore {
       allow list: if isAdmin() || isJuror();
 
       // If applications are open
-      // allow create: if isSignedIn();
-      // allow update: if isOwner();
+      allow create: if isSignedIn();
+      allow update: if isOwner();
```
Comment (constancecchen) [inline firestore.rules]: Awesome catch. As a heads up, I don't think I have firebase access anymore, so I can't test the app itself for you (sorry!), so I'm not sure if there's any other tricky shenanigans like this.

Hopefully not, but one caveat I do remember from last year - I mostly used this `firebase.rules` files as version control reference. You can update the rules themselves directly from Firebase's dashboard/GUI, and I remember doing so frequently. You might want to do double check that the prod and dev database rules have indeed been updated on deploy, and if not, manually copy and paste this file instead.

File: _data/social-media.yml (hunk 1 above)
Comment (aschearer) [reply _data/social-media.yml]: Ah, good to know. I think it's OK if people RSVP now :-)

File: _data/schedule.json (hunk 4 above)
Comment (aschearer) [reply _data/schedule.json]: You're right. Would you happen to remember what this bool does?

Comment (aschearer) [reply _data/schedule.json]: Site looked right with it true... maybe it only matters when applicationsOpen is false.

File: _years/2018/index.md (hunk 5 above)
Comment (aschearer) [reply _years/2018/index.md]: *Hugs*

File: firestore.rules (hunk 6 above)
Comment (aschearer) [reply firestore.rules]: Eeee OK thank you for the warning. I was able to do what I expected against the dev-site -- register a game, admin. I think it's working *fingers crossed*.

File: _pages/about.html (hunk 3 above)
Comment (aschearer) [reply _pages/about.html]: I will if you want, but I think you deserve credit for all your past work :-)

File: _includes/svgs/robot7.svg (hunk 2 above)
Comment (aschearer) [reply _includes/svgs/robot7.svg]: Sarah made them!

**Comment by aschearer** [comment]
Merged into master. Will start a new pull request for any follow up changes.

**Pull request merged**